android/src/main/java/bbct/android/common/activity/MainActivity.java

```diff
@@ -167,17 +167,29 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        int itemId = item.getItemId();
-
-        if (itemId == R.id.about_menu) {
-            this.getSupportFragmentManager()
-                    .beginTransaction()
-                    .replace(R.id.fragment_holder, new About(), FragmentTags.ABOUT)
-                    .addToBackStack(FragmentTags.ABOUT)
-                    .commit();
-            return true;
+        int menuId = item.getItemId();
+
+        switch (menuId) {
```

Comment:
Thanks for changing this to a `switch`! When I originally added the Lite and Premium versions of the app, I had to implement this as an `if` statement because the `R.id` constants were not declared as `final`.